nix/default.nix

```diff
@@ -0,0 +1,63 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
i think we should also pin `nixpkgs`, as you already did for `pypi2nix`. but nixpkgs is a bit different to pin. you can find instructions here https://nixos.wiki/wiki/How_to_fetch_Nixpkgs_with_an_empty_NIX_PATH

Reply:
Good idea!